From: Hariprasad Shenai <hariprasad@chelsio.com>
Date: Wed, 12 Nov 2014 15:01:06 +0530
Subject: [PATCH 118/230] cxgb4: Fix static checker warning
Patch-mainline: v3.19-rc1
Git-commit: c94f8f4156c6ea348bf1c54075ec84ade62ba7c2
References: bsc#924378 FATE#318565 bsc#921338

Fix static checker warning that got introduced in commit e2ac9628959cc152
("cxgb4: Cleanup macros so they follow the same style and look consistent, part
2") due to accidental checkin of bogus line.

Signed-off-by: Hariprasad Shenai <hariprasad@chelsio.com>
Signed-off-by: David S. Miller <davem@davemloft.net>
Acked-by: David Chang <dchang@suse.com>
---
 drivers/net/ethernet/chelsio/cxgb4/cxgb4_main.c |    1 -
 1 file changed, 1 deletion(-)

--- a/drivers/net/ethernet/chelsio/cxgb4/cxgb4_main.c
+++ b/drivers/net/ethernet/chelsio/cxgb4/cxgb4_main.c
@@ -5753,7 +5753,6 @@ static int adap_init0(struct adapter *ad
 					    "No Configuration File present "
 					    "on adapter. Using hard-wired "
 					    "configuration parameters.\n");
-					goto bye;
 					ret = adap_init0_no_config(adap, reset);
 				}
 			}
